package.json

```diff
@@ -7,6 +7,7 @@
     "analyze": "webpack-bundle-analyzer build/vagovprod/generated/stats.json",
     "build": "./script/build.sh",
     "build-analyze": "NODE_ENV=production npm run build -- --buildtype vagovprod --analyzer; npm run analyze",
+    "build:compare": "node script/content-build-compare.js && diff --color websiteContentBuildHash.txt standaloneContentBuildHash.txt",
```

Comment:
I'm not thrilled by how the node script writes the files and we have a separate bash command to do the diffing, but I wasn't able to get the output to show if I ran the diff inside of the node script.

Reply:
Still unsuccessful :disappointed:

Reply:
This is totally fine- we're not going to be keeping it around once everything is separated and validated.